packs/elastic/scripts/logsfeed.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
Comments from the previous PR still apply - please move additional scripts and files into `etc/` directory.